### PR TITLE
Add repo on RHEL8 (Ansible system roles)

### PR DESCRIPTION
**Pull request by Lennonka**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2121848

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3

### CODE REVIEW

File: guides/common/modules/proc_adding-rhel-system-roles.adoc [hunk 1]
```diff
@@ -13,18 +13,23 @@ Before subscribing to the Extras channels, see the https://access.redhat.com/sup
 
 .Procedure
```
Comment (maximiliankolb) [inline guides/common/modules/proc_adding-rhel-system-roles.adoc:15]:
```suggestion
```

File: guides/common/modules/proc_adding-rhel-system-roles.adoc [hunk 2]
```diff
@@ -13,18 +13,23 @@ Before subscribing to the Extras channels, see the https://access.redhat.com/sup
...
 ----
+* On {RHEL} 8, ensure that the Appstream repository is enabled.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos --enable=rhel-8-for-_architecture_-appstream-rpms
+----
 +
+You must enable an Appstream repository that is designated for your architecture.
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/appendix_rhel-8-repositories_upgrading-from-rhel-7-to-rhel-8[RHEL 8 repositories].
```
Comment (maximiliankolb) [inline guides/common/modules/proc_adding-rhel-system-roles.adoc:32]: Please make this satellite-only.

File: guides/common/modules/proc_adding-rhel-system-roles.adoc [hunk 3]
```diff
@@ -12,19 +12,24 @@ For up-to-date information about support levels and general information about {R
...
 .Procedure
-
-ifdef::satellite[]
-. Ensure that the `rhel-7-server-extras-rpms` repository is enabled.
-endif::[]
-ifndef::satellite[]
-. On {RHEL}, ensure that the `rhel-7-server-extras-rpms` repository is enabled.
-endif::[]
+. Ensure that the following repository is enabled:
+* On {RHEL} 7, ensure that the *Extras* repository is enabled.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_adding-rhel-system-roles.adoc:16]:
```suggestion
* On {RHEL} 7, ensure that the *Extras* repository is enabled:
```

File: guides/common/modules/proc_adding-rhel-system-roles.adoc [hunk 4]
```diff
@@ -12,19 +12,24 @@ For up-to-date information about support levels and general information about {R
...
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # subscription-manager repos --enable=rhel-7-server-extras-rpms
 ----
+* On {RHEL} 8, ensure that the *Appstream* repository is enabled.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_adding-rhel-system-roles.adoc:22]:
```suggestion
* On {RHEL} 8, ensure that the *Appstream* repository is enabled:
```

**Review by maximiliankolb — APPROVED** [review]
Two small suggestions, LGTM.

File: guides/common/modules/proc_adding-rhel-system-roles.adoc (hunk 3 above)
Comment (Lennonka) [reply guides/common/modules/proc_adding-rhel-system-roles.adoc:16]: Good catch, thanks!

**Comment by Lennonka** [comment]
Cherry-picked:
- 875974b50..caba9cfda  3.4 -> 3.4
- 1063a3b09..0e7df03ab  3.3 -> 3.3
- a5612fa64..f96f63ca7  3.2 -> 3.2
- cc91437c5..f3d2a269d  3.1 -> 3.1

**Pull request merged**